backend/checkmach.ml

```diff
@@ -1489,6 +1492,39 @@ end = struct
       Misc.fatal_errorf "Unexpected duplicate annotation %a for %s"
         Debuginfo.print_compact dbg fun_name ()
 
+  let of_cfg codegen_options spec fun_name dbg =
+    let a =
+      List.filter_map
+        (fun (c : Cfg.codegen_option) ->
```

Comment:
It is, but with a [different type](https://github.com/ocaml-flambda/flambda-backend/blob/b160d500a9234089dbbd1c132c58be55086f051c/backend/checkmach.ml#L1392-L1396). I didn't know how to avoid the duplication, is there a trick with `'a` types?